src/server/server.cc

```diff
@@ -846,13 +846,18 @@ void Server::GetRocksDBInfo(std::string *info) {
   db->GetAggregatedIntProperty("rocksdb.num-live-versions", &num_live_versions);
 
   string_stream << "# RocksDB\r\n";
+
+  uint64_t block_cache_usage = 0;
+  // All column families share the same block cache, so it's good to count a single one.
+  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
+  db->GetIntProperty(cf_handle, "rocksdb.block-cache-usage", &block_cache_usage);
+  string_stream << "block_cache_usage:" << block_cache_usage << "\r\n";
+
   for (const auto &cf_handle : *storage->GetCFHandles()) {
-    uint64_t estimate_keys = 0, block_cache_usage = 0, block_cache_pinned_usage = 0, index_and_filter_cache_usage = 0;
+    uint64_t estimate_keys = 0, block_cache_pinned_usage = 0, index_and_filter_cache_usage = 0;
```

Comment:
Quick question: would `block_cache_pinned_usage` be shared?

Reply:
Can you split each identifier in a dedicated statement
```suggestion
    uint64_t estimate_keys = 0;
    uint64_t block_cache_pinned_usage = 0;
    uint64_t index_and_filter_cache_usage = 0;
```
https://sonarcloud.io/project/issues?open=AY7Xa5ShOhsi8ezKPlrb&id=apache_kvrocks